.travis.yml

```diff
@@ -0,0 +1,34 @@
+# Config file for automatic testing at travis-ci.org
+
+sudo: false
+language: python
+
+matrix:
+    include:
+      - python: 2.7
+        env: TOX_ENV=py27
+      - python: 3.4
+        env: TOX_ENV=py34
+      - python: 3.5
+        env: TOX_ENV=py35
```

Comment:
tcms-api pacakge requires Python 3.6+ because that fixes issues in the XML-RPC transport layer so unless you implement all the RPC transport yourself there's no way to support older versions of Python.